install/lib/class.FroxlorInstall.php

```diff
@@ -1273,6 +1298,48 @@ private function _guessWebserver()
 		}
 	}
 
+
+	/**
+	 * get/guess linux distribution
+	 */
+	private function _guessDistribution()
```

Comment:
too hardcoded... why not check for supported distros (from xml, see https://github.com/Froxlor/Froxlor/blob/master/admin_configfiles.php#L131) and check $os_version data against that to see if it's a match?

Reply:
sure